app/uk/gov/hmrc/economiccrimelevyregistration/models/addresslookup/AlfServiceLabels.scala

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.economiccrimelevyregistration.models.addresslookup
+
+import play.api.i18n.{Lang, MessagesApi}
+import play.api.libs.json.{Json, OFormat}
+import uk.gov.hmrc.economiccrimelevyregistration.config.AppConfig
+import uk.gov.hmrc.economiccrimelevyregistration.controllers.routes
+import uk.gov.hmrc.hmrcfrontend.views.config.StandardBetaBanner
+
+final case class AlfEnCyLabels(en: AlfLabels, cy: AlfLabels)
+
+object AlfEnCyLabels {
+  def apply(appConfig: AppConfig)(implicit messagesApi: MessagesApi): AlfEnCyLabels = {
+    def betaBanner(lang: Lang): String = new StandardBetaBanner()
+      .apply(appConfig.feedbackUrl(routes.IsUkAddressController.onPageLoad().url))(messagesApi.preferred(Seq(lang)))
+      .content
+      .asHtml
+      .body
+
+    AlfEnCyLabels(
+      en = AlfLabels(
+        appLevelLabels = AlfAppLabels(
+          navTitle = messagesApi("service.name")(Lang("en")),
+          phaseBannerHtml = betaBanner(Lang("en"))
+        ),
+        countryPickerLabels = AlfCountryPickerLabels(
+          submitLabel = messagesApi("alf.labels.submit")(Lang("en"))
+        ),
+        selectPageLabels = AlfSelectPageLabels(
+          title = messagesApi("alf.labels.select.title")(Lang("en")),
+          heading = messagesApi("alf.labels.select.heading")(Lang("en")),
+          submitLabel = messagesApi("alf.labels.submit")(Lang("en"))
+        ),
+        lookupPageLabels = AlfLookupPageLabels(
+          title = messagesApi("alf.labels.lookup.title")(Lang("en")),
+          heading = messagesApi("alf.labels.lookup.heading")(Lang("en")),
+          postcodeLabel = messagesApi("alf.labels.lookup.postcode")(Lang("en")),
+          submitLabel = messagesApi("alf.labels.submit")(Lang("en"))
+        ),
+        editPageLabels = AlfEditPageLabels(
+          title = messagesApi("alf.labels.edit.title")(Lang("en")),
+          heading = messagesApi("alf.labels.edit.heading")(Lang("en")),
+          submitLabel = messagesApi("alf.labels.submit")(Lang("en"))
+        )
+      ),
+      cy = AlfLabels(
+        appLevelLabels = AlfAppLabels(
+          navTitle = messagesApi("service.name")(Lang("cy")),
```

Comment:
we are using `Lang("cy")` and `Lang("en")` in quite a lot of places now (including GRS), maybe it makes sense to put this in a `Languages` object or something as vals `english` and `welsh` or something?